test/e2e-tests.sh

```diff
@@ -224,18 +224,23 @@ EOF
 function install_catalogsource(){
   header "Installing CatalogSource"
 
-  tag_serverless_operator_image
-
-  local serverless_image=$INTERNAL_REGISTRY/$OPERATORS_NAMESPACE/$SERVERLESS_OPERATOR
-  ./hack/catalog.sh | sed -e "s+\(.* containerImage:\)\(.*\)+\1 ${serverless_image}+g" > catalogsource-ci.yaml
-  oc apply -n $OPERATORS_NAMESPACE -f catalogsource-ci.yaml || return 1
+  local serverless_image=$(tag_serverless_operator_image)
+  if [[ -n "${serverless_image}" ]]; then
+    ./hack/catalog.sh | sed -e "s+\(.* containerImage:\)\(.*\)+\1 ${serverless_image}+g" > $CATALOG_SOURCE_FILENAME
+  else
+    ./hack/catalog.sh > $CATALOG_SOURCE_FILENAME
+  fi
+  oc apply -n $OPERATORS_NAMESPACE -f $CATALOG_SOURCE_FILENAME || return 1
 
   header "CatalogSource installed successfully"
 }
 
 function tag_serverless_operator_image(){
-  oc policy add-role-to-group system:image-puller system:serviceaccounts:${OPERATORS_NAMESPACE} --namespace=${OPENSHIFT_BUILD_NAMESPACE}
-  oc tag --insecure=false -n ${OPERATORS_NAMESPACE} ${OPENSHIFT_REGISTRY}/${OPENSHIFT_BUILD_NAMESPACE}/stable:${SERVERLESS_OPERATOR} ${SERVERLESS_OPERATOR}:latest
+  if [[ -n "${OPENSHIFT_BUILD_NAMESPACE}" ]]; then
+    oc policy add-role-to-group system:image-puller system:serviceaccounts:${OPERATORS_NAMESPACE} --namespace=${OPENSHIFT_BUILD_NAMESPACE}
+    oc tag --insecure=false -n ${OPERATORS_NAMESPACE} ${OPENSHIFT_REGISTRY}/${OPENSHIFT_BUILD_NAMESPACE}/stable:${SERVERLESS_OPERATOR} ${SERVERLESS_OPERATOR}:latest
```

Comment:
Looks like the output of these lines needs to be sent to /dev/null so that we don't get the error in logs:
`+ sed -e 's+\(.* containerImage:\)\(.*\)+\1 Tag serverless-operator:latest set to registry.svc.ci.openshift.org/ci-op-bqg1pvq8/stable:serverless-operator.
image-registry.openshift-image-registry.svc:5000/openshift-operators/serverless-operator+g'`

Reply:
fixed